components/ee/ws-scheduler/pkg/scheduler/metrics/metric-recorder.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Licensed under the Gitpod Enterprise Source Code License,
+// See License.enterprise.txt in the project root folder.
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type MetricRecorder interface {
+	Inc()
+	Dec()
+	Clear()
+}
+
+type PendingPodsRecorder struct {
+	recorder prometheus.Gauge
+}
+
+func NewActivePodsRecorder() *PendingPodsRecorder {
+	return &PendingPodsRecorder{
+		recorder: PendingPods.WithLabelValues("queue", "active"),
+	}
+}
+
+func NewBackoffPodsRecorder() *PendingPodsRecorder {
+	return &PendingPodsRecorder{
+		recorder: PendingPods.WithLabelValues("queue", "backoff"),
+	}
+}
```

Comment:
I think it was just a mistake, but why do we hardcode `"queue"` here? 
This results in every `gitpod_ws_scheduler_pending_pods` with the same queue label value:
![image](https://user-images.githubusercontent.com/24193764/108842290-cfc21480-75b7-11eb-9cce-269c2c7ec6e2.png)

If it was not a mistake and all metrics should have this hardcoded value, I don't see a reason to keep it :thinking:

Reply:
Maybe you wanted to put the second string, `active` and `backoff` as the first argument and regular/ghost/etc as the second argument?

Reply:
Thx, that's indeex a copy'n'paste issue. `backoff` | `active` are the values for `queue`.

Reply:
Fixed by removing `workspaceType` entirely. It was not possible to retain it easily, and it was added merely because I could, not because it offers too valueable insights. :man_shrugging: